src/pages/info/index.tsx

```diff
@@ -1,7 +1,15 @@
 import { InfoPageLayout } from 'views/Info'
 import Overview from 'views/Info/Overview'
+import { useRouter } from 'next/router'
+import { ChainId } from '@pancakeswap/sdk'
 
 const InfoPage = () => {
+  const router = useRouter()
+  const { chainId, chainName } = router.query
+  if (chainId === ChainId.ETHEREUM.toString() && !chainName)
+    router.replace({ pathname: '/info/eth', query: '' }, undefined, { shallow: true })
+  else if (!chainName && chainId === ChainId.BSC.toString())
+    router.replace({ pathname: '/info', query: '' }, undefined, { shallow: true })
```

Comment:
can get the chainId and useEffect once

Reply:
useActiveChainId or useActiveWeb3React
`sesssionChainId -> queryChainId -> wallet.chain`